cogs/shop.py

```diff
@@ -847,10 +847,20 @@ def scrollEmbedCheck(r, u):
                     
 
                 if bookChoice == "Spellbook":
-                    if 'Wizard' not in bRecord['Classes']:
-                        await channel.send(f"**{bRecord['Name']}*** is not a Wizard spell that can be copied into your spellbook.")
-                        ctx.command.reset_cooldown(ctx)
-                        return   
+                    f 'Wizard' not in bRecord['Classes']:
+                        if "Race" in bRecord:   # Determines if the spell is a Mark of X spell
+                            if "Mark" not in charRecords['Race']:   # Determines if the race is a Mark of X race
+                                await channel.send(f"***{bRecord['Name']}*** is not a Wizard spell that can be copied into your spellbook.")
+                                ctx.command.reset_cooldown(ctx)
+                                return
+                            elif charRecords['Race'] not in bRecord['Race']:    # Determines if the Mark of X race is a valid Mark of X race for the spell
+                                await channel.send(f"***{bRecord['Name']}*** is not a Wizard spell that can be copied into your spellbook.")
+                                ctx.command.reset_cooldown(ctx)
+                                return
+                        else:
+                            await channel.send(f"***{bRecord['Name']}*** is not a Wizard spell that can be copied into your spellbook.")    # If the spell isn't a wizard spell, bot checks to see if it is a mark spell.
+                            ctx.command.reset_cooldown(ctx)
+                            return  
```

Comment:
Could you please refactor these so that the message text is not duplicated 3 times